diagnostic.md

```diff
@@ -44,9 +58,23 @@ How would we reference each of the following categories of elements?
 -   Only those elements belonging to both the `important` and `red` _classes_
 -   All `p` elements whose parent elements are `div`s
 
-<!-- your answer starts here -->
+```CSS
+.big{
+}
+#id{
 
-<!-- your answer ends here -->
+}
+.important, .red{
```

Comment:
This will select everything that has either ```important``` or ```red``` as a class.  To select elements that have both ```important``` and ```red``` as classes, use ```.important.red```.